index.js

```diff
@@ -236,7 +341,7 @@ export function lfind(str, i) {
       if (i < 2) {
         search = false;
       }
-      if (/[a-z]|[A-Z]|[0-9]|[.]|[-]/.test(str[i - 1])) {
+      if (/[a-zA-Z]|[0-9]|[.]|[-]|[\s]/.test(str[i - 1])) {
```

Comment:
Just one set of brackets, here and several other places.  (Search for ]|[ ).

Reply:
Fixed